packages/create-svelte/test/check.js

```diff
@@ -133,7 +133,10 @@ for (const template of templates) {
 
 		for (const script of scripts_to_test) {
 			const tests = script_test_map.get(script) ?? [];
-			tests.push([`${template}-${types}`, () => exec_async(`pnpm ${script}`, { cwd })]);
+			tests.push([
+				`${template}-${types}`,
+				() => exec_async(`ESLINT_USE_FLAT_CONFIG=false pnpm ${script}`, { cwd })
```

Comment:
This is a workaround for ESLint configs in `.test-tmp/create-svelte` inheriting from the main ESLint config. I expect to be able to remove it when `create-svelte` is migrated to ESLint 9 too (sveltejs/kit#12089).

Reply:
should be able to rebase and update this now

Reply:
Done.